src/api/kube.ts

```diff
@@ -1199,6 +1199,19 @@ export class KubeHelper {
     }
   }
 
+  async deleteDeployment(namespace: string, name: string): Promise<boolean> {
```

Comment:
I don't see if this func is used anywhere.

Reply:
Well, yes, I don't use it anymore in my PR, but this functionality is missing in our "lib" for communication with the cluster, so decided to leave for future use. It doesn't harm.

Reply:
Pls return Promise<void>`

Reply:
ok, will return `void`